assets/panel/debugger.properties

```diff
@@ -55,6 +59,10 @@ pausePendingButtonTooltip=Waiting for next execution
 # button when the debugger is in a paused state.
 resumeButtonTooltip=Resume %S
 
+# LOCALIZATION NOTE (resumeButtonTooltip): The label that is displayed for the dropdown 
```

Comment:
Please double check that the string ID referenced in the comment matches the actual string (it doesn't here).